src/components/common/DeleteButton.tsx

```diff
@@ -46,7 +46,10 @@ const DeleteButton = ({
     onClose?.();
   };
 
-  const text = translateBuilder(BUILDER.DELETE_BUTTON);
+  const text =
+    items.length > 1
+      ? translateBuilder(BUILDER.DELETE_ALL_SELECTED_ITEMS_BUTTON)
+      : translateBuilder(BUILDER.DELETE_BUTTON);
```

Comment:
I think you could use the plurals of i18next here (https://www.i18next.com/translation-function/plurals)